components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -7,26 +7,79 @@ documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
 
 When adding a new metric, the workflow is:
 * Decide on which metric type you want to use.
-* Add a new entry to metrics.yaml.
-* Add code to your project to record into the metric by calling Glean.
+* Add a new entry to `metrics.yaml`.
+* Add code to your project to record into the metric by calling glean.
 
 **Important**: as stated [here](../../README.md#before-using-the-library), any new data collection requires
 documentation and data-review. This is also required for any new metric automatically collected
 by glean.
 
+# The `metrics.yaml` file
+
+The `metrics.yaml` file defines the metrics your application or library will
+send. They are organized into categories. Category names may optionally contain
+`.`'s to indicate subcategories. 
+
+Therefore, the overall organization is:
+
+```YAML
+# Required to indicate this is a `metrics.yaml` file
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+category1:
+  metric1:
+    type: string
+    description: |
+      An example metric
+    notification_emails:
+      - CHANGE-ME@test-only.com
+    bugs:
+      - 123456789
+    data_reviews:
+      - http://test-only.com/path/to/data-review
+    expires:
+      - 2100-01-01
```

Comment:
Definitely use a sooner date or something that doesn’t build here to avoid copy-pasting of this far-off date.

Reply:
This is a hard one.  If we make it a sooner date, then copying-and-pasting will result in an already disabled event.  Probably have to use a comment here to avoid confusion.

Reply:
Yeah, maybe... an expired date and a comment to change it?